subsys/net/ip/Kconfig

```diff
@@ -156,7 +156,7 @@ config NET_TCP_RETRY_COUNT
 	The following formula can be used to determine the time (in ms)
 	that a segment will be be buffered awaiting retransmission:
 	n=NET_TCP_RETRY_COUNT
-	∑((1<<n) * 200)
+	Sigma((1<<n) * 200)
```

Comment:
I'd say this should be replaced with "Sum" instead. (And overall, that "formula" looks pretty weird on its own, an attempt to render a mathematical notation in ascii, it would be better replaced with a more textual rendition.

Reply:
Sigma to Sum change made.